deploy/helm/kubecf/assets/operations/instance_groups/app_autoscaler.yaml

```diff
@@ -0,0 +1,1150 @@
+  value:
+    domain: autoscalerpostgres.service.cf.internal
+    targets:
+    - deployment: cf
+      domain: bosh
+      instance_group: postgres_autoscaler
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: apiserver.service.cf.internal
+    targets:
+    - deployment: cf
+      domain: bosh
+      instance_group: asapi
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: autoscalerscheduler.service.cf.internal
+    targets:
+    - deployment: cf
+      domain: bosh
+      instance_group: asactors
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: servicebroker.service.cf.internal
+    targets:
+    - deployment: cf
+      domain: bosh
+      instance_group: asapi
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: eventgenerator.service.cf.internal
+    targets:
+    - deployment: cf
+      domain: bosh
+      instance_group: asmetrics
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: scalingengine.service.cf.internal
+    targets:
+    - deployment: cf
+      domain: bosh
+      instance_group: asactors
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: metricsgateway.service.cf.internal
+    targets:
+    - deployment: cf
+      domain: bosh
+      instance_group: asnozzle
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: metricsserver.service.cf.internal
+    targets:
+    - deployment: cf
+      domain: bosh
+      instance_group: asmetrics
+      network: default
+      query: '*'
+
+# add autoscaler related releases
+- type: replace
+  path: /releases/-
+  value:
+    name: app-autoscaler
+    version: "2.0.0"
+    url: "https://bosh.io/d/github.com/cloudfoundry-incubator/app-autoscaler-release?v=2.0.0"
+    sha1: "b2bda4bd9ff9b902fa6a6f3d37c13b681b3402b0"
+- type: replace
+  path: /releases/-
+  value:
+    name: postgres
+    version: "25"
+    url: "https://bosh.io/d/github.com/cloudfoundry/postgres-release?v=25"
+    sha1: "20929ee4b0c64fd97072a266311a6d00714124a7"
+
+# add autoscaler instance group
+- type: replace
+  path: /instance_groups/-
+  value:
+    name: postgres_autoscaler
+    azs:
+    - z1
+    instances: 1
+    update:
+      serial: true
+    stemcell: default
+    vm_type: small
```

Comment:
`vm_type` is not relevant with cf-operator.

Reply:
The `vm_type` value and below was removed.